doc/radosgw/layout.rst

```diff
@@ -37,15 +37,15 @@ You can use the following commands to inspect metadata entries: ::
     $ radosgw-admin metadata get bucket.instance:<bucket>:<bucket_id>
     $ radosgw-admin metadata get user:<user>   # get or set
     
-Some variables have been used in above commands, they are:
+Variables have been used in above commands, they are:
 
-- _user_: Holds user information
-- _bucket_: Holds a mapping between bucket name and bucket instance id
-- _bucket.instance_: Holds bucket instance information[2]
+- ``user``_: Holds user information
+- ``bucket``_: Holds a mapping between bucket name and bucket instance id
+- ``bucket.instance``_: Holds bucket instance information[2]
```

Comment:
i think the trailing _ is causing this to render wrong in https://ceph--62979.org.readthedocs.build/en/62979/radosgw/layout/#metadata

Reply:
Honestly I'm still trying to grok those underscores.  I pushed a removal.